### PR TITLE
fix(net6): Restore partial support for MUX progressring

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Show the native progress ring in place of the MUX progressring.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by bbhxwl** [comment]
> 

I have never successfully run uno program in gitpod.

**Comment by jeromelaban** [comment]
> I have never successfully run uno program in gitpod.

Avoid commenting on PRs, it's best if you open a discussion. Yet make sure to read this first: https://platform.uno/docs/articles/features/working-with-gitpod.html

File: src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj [hunk 1]
```diff
@@ -118,6 +118,7 @@
 	
 	<ItemGroup>
 	  <Folder Include="MacOS\" />
+	  <Folder Include="obj\" />
```
Comment (MartinZikmund) [inline src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj:121]: Is this needed?

**Review by MartinZikmund — APPROVED** [review]

**Review by agneszitte-nventive — APPROVED** [review]

File: src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj (hunk 1 above)
Comment (jeromelaban) [reply src/SamplesApp/SamplesApp.net6mobile/SamplesApp.net6mobile.csproj:121]: No, but I'll remove it in the next PR. Thanks!

**Pull request merged**